package.json

```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "@clevercloud/client": "^8.1.2",
     "clf-date": "^0.2.0",
-    "cliparse": "^0.3.3",
+    "cliparse": "file:cliparse-0.3.4.tgz",
```

Comment:
Shouldn't we made a proper update of cliparse?

Reply:
Yes, of course, This PR is in a state that lets you check the preview. Once validated, I will make the `cliparse` release and include it properly in the package.json.

Reply:
For the QA which commands are impacted by this change? All?

Reply:
This is about bad commands: 
- unknown command should exit with 1
- non-terminated command should exit 1,
- command with missing required arg/option should exit 1,
- command with unknown arg/option should exit 1,
- command with malformed arg/option should exit 1.

Reply:
Ok I see, for instance: 

```
clever wrong_command
./clever addon list --format wrong
```

Shouldn't we then plot an `Error` message? as `[ERROR] format: The output format must be one of human, json`?

Reply:
https://github.com/CleverCloud/clever-tools/issues/742